### DISCUSSION TITLE
Countdown timer.

### DISCUSSION THREAD

**Discussion by JamieBort** [wger-project/wger#900] [Ideas]
I am new to wger and am VERY pleased to have found a gym app with a public api. And I am even more ecstatic that there is a count up clock between sets.
That said, it would be very much appreciated to have a new feature that counts down between sets. And then audibly notifies me when 30, 60, or 90 seconds have passed.
Thank you very much.

**Comment by rolandgeider** [comment]
Hi! Glad that you like it!! The timer is there as a really simple way to track the pauses between sets, I didn't want to add much logic there since everybody trains differently. That being said, having a countdown can be useful as well (and in case somebody wants to work on this, just ping me)

**Reply by JamieBort** [reply]
I may circle around to create that feature. Unfortunately I cannot commit for a few months.
In the meantime, the current timer is just fine. Thanks!

**Reply by JamieBort** [reply]
FWIW, I see that I could have posted the above in the "Suggest a feature" section here:
https://github.com/wger-project/wger/issues/new/choose
I searched for it (unsuccessfully) via GitHub. And came across it just now via https://wger.de/en/software/about-us
Apparently there are many aspects of GitHub that I still need to become familiar with. Sorry about that!

**Reply by rolandgeider** [reply]
Don't worry about it! Discussions can also be used to suggest new features and can be easily referenced from issues as well

**Comment by clafalco** [comment]
Hello, it would be very nice if there was a timer with a sound start and pause in the timed exercises
